### PR TITLE
Add Seperator

### PR DESCRIPTION
**Pull request by Maan2003**

### CODE REVIEW

File: src/seperator.rs [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    pub fn set_size(&mut self, size: impl Into<KeyOrValue<f64>>) {
+        self.size = size.into();
+    }
+
+    pub fn with_color(mut self, color: impl Into<KeyOrValue<Color>>) -> Self {
+        self.color = color.into();
+        self
+    }
+
+    pub fn with_stroke_style(mut self, stroke_style: StrokeStyle) -> Self {
```
Comment (Finnerale) [inline src/seperator.rs:63]: `set_stroke_style` seems to be  missing.

**Review by Finnerale — APPROVED** [review]
Seems like a good addition! One setter is missing though.

File: src/seperator.rs (hunk 1 above)
Comment (Maan2003) [reply src/seperator.rs:63]: oh I forgot that. adding...

**Review by derekdreery — APPROVED** [review]
Awesome, I'm excited to see how this gets used and what people feel about it.

**Pull request merged**